azbfs/url_file.go

```diff
@@ -17,6 +17,16 @@ type FileURL struct {
 	path           string
 }
 
+//rename me?
+// BlobFSFileHTTPHeaders represents the set of custom headers available for defining information about the content.
+type BlobFSFileHTTPHeaders struct {
```

Comment:
If you'd like alternative name suggestions, then looking at the corresponding types in the File and Blob APIs, I'd suggest this one be named BlobFsHTTPHeaders.  (Technically, BlobFSHTTPHeaders would the "correct" capitalization, but that's very hard to read).